common/persistence/dataInterfaces.go

```diff
@@ -688,6 +688,7 @@ type (
 		SignalInfoCount        int
 		SignalRequestIDCount   int
 		BufferedEventsCount    int
+		TaskCountByCategory    map[string]int
```

Comment:
`map[tasks.Category]int`?

Reply:
I picked the name because it's slightly smaller than the whole struct, but I don't feel too strongly about it.